File: Model.Tests/TeamTests.vb
<TestClass()> _
Public Class TeamTests

    <TestMethod()> _
    Public Sub TeamTest()
        ' Arrange
        Dim number As Integer = 3

        ' Act
        Dim result = New Team(number)

        ' Assert
        Assert.AreEqual("Team N°3", result.Name, "Unexpected Name")
    End Sub

End Class

File: POFF.Kicker.Tests/TeamTests.cs
using NUnit.Framework;
using POFF.Kicker.Domain;

namespace POFF.Kicker.Tests;

[TestFixture]
public class TeamTests
{

    [Test]
    public void TeamTest()
    {
        // Arrange
        int number = 3;

        // Act
        var result = new Team(number);

        // Assert
        Assert.AreEqual("Team N°3", result.Name, "Unexpected Name");
    }

}